### PR TITLE
add news items for new releases 7.8.7 and 8.0.1

### PR DESCRIPTION
**Pull request by veroandreo**
This is a draft PR adding news items for the latest releases. The 7.8.7 news is ready, the 8.0.1 one needs a short description of changes and a screenshot. Also, please check links; I added links for windows but not sure if they are ready yet

This is how the 7.8.7 news looks like:
image

### CODE REVIEW

**Comment by landam** [comment]
Related to Windows standalone installer:

* 8.0.1 ready to be tested: https://grass.osgeo.org/grass80/binary/mswindows/native/WinGRASS-8.0.1-1-Setup.exe
* 7.8.7 64bit ready to be tested: https://grass.osgeo.org/grass78/binary/mswindows/native/x86_64/WinGRASS-7.8.7-1-Setup-x86_64.exe
* 7.8.7 32bit is not currently built and most probably will be not prepared by me (I have no 32bit build environment currently)

File: content/news/2022_02_23_GRASS_GIS_7_8_7_released.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+The overview of features in the 7.8 release series is available at
+[new features in GRASS GIS 7.8](https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78).
+See also our detailed announcement with the full list of changes and
+bugs fixed at
+[https://github.com/OSGeo/grass/releases/tag/7.8.7](https://github.com/OSGeo/grass/releases/tag/7.8.7).
+
+**Binaries/Installer download:**
+
+- winGRASS 7.8.7/standalone:
+  [32bit standalone installer](/grass78/binary/mswindows/native/x86/WinGRASS-7.8.7-1-Setup-x86.exe) \| [64bit standalone installer](/grass78/binary/mswindows/native/x86_64/WinGRASS-7.8.7-1-Setup-x86_64.exe)
```
Comment (landam) [inline content/news/2022_02_23_GRASS_GIS_7_8_7_released.md:33]:
```suggestion
[64bit standalone installer](/grass78/binary/mswindows/native/x86_64/WinGRASS-7.8.7-1-Setup-x86_64.exe)
```

File: content/news/2022_02_23_GRASS_GIS_7_8_7_released.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- winGRASS 7.8.7/OSGeo4W:
+  [32bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86-v1.exe) \| [64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe)
```
Comment (landam) [inline content/news/2022_02_23_GRASS_GIS_7_8_7_released.md:35]:
```suggestion
[64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe)
````

File: content/news/2022_02_23_GRASS_GIS_7_8_7_released.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+   title="GRASS GIS 7.8.7 GUI in dark mode"
+   width="45%" style="float:right;padding-left:10px;padding-top:20px">
+</a>
+
+As a follow-up to the previous GRASS GIS 7.8.6 we have published the
+**new release GRASS GIS 7.8.7** with more than 40 fixes mostly focused
+on [g.extension](https://grass.osgeo.org/grass78/manuals/g.extension.html) 
+and Python 3.10 related errors in the GUI. Furthermore,
+changes to the *libgis* allowing to recover the last changed date in core
+modules and addons' manual pages were backported from newer versions.
```
Comment (neteler) [inline content/news/2022_02_23_GRASS_GIS_7_8_7_released.md:22]:
```suggestion
on the addon installation module [g.extension](https://grass.osgeo.org/grass78/manuals/g.extension.html) 
and Python 3.10 related errors in the graphical user interface (GUI). Furthermore,
changes to the internal *libgis* allowing to recover the last changed date in the manual pages
of core modules and addons has been backported from newer GRASS GIS versions.
```

... minor wording suggestions.

File: content/news/2022_02_24_GRASS_GIS_8_0_1_released.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+#### Binaries/Installers download
+
+- winGRASS 8.0.1/standalone:
+  [64bit standalone installer](/grass80/binary/mswindows/native/WinGRASS-8.0.1-1-Setup.exe)
+- winGRASS 8.0.1/OSGeo4W:
+  [64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/v2/osgeo4w-setup.exe)
+- Mac
+    - [Official Application Bundle](http://grassmac.wikidot.com/downloads)
+    - [MacPorts](https://ports.macports.org/port/grass/)
```
Comment (neteler) [inline content/news/2022_02_24_GRASS_GIS_8_0_1_released.md:30]:
```suggestion
    - [MacPorts](https://ports.macports.org/port/grass/)
```

This link says:
```
grass v 6.4.6
Obsolete port, replaced by grass7
```

Perhaps to be removed? Or does e.g. @nilason have an idea how to get a fresh macports version?

File: content/news/2022_02_24_GRASS_GIS_8_0_1_released.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+- Linux
+    - [Debian](https://tracker.debian.org/pkg/grass)
+    - [Fedora/Centos/EPEL](https://src.fedoraproject.org/rpms/grass)
+    - [Ubuntu](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable/+packages?field.name_filter=grass)
+    - ... further binary packages for other Linux distributions will follow shortly, please check at [software downloads](/download/software/index.html#g78x)
```
Comment (neteler) [inline content/news/2022_02_24_GRASS_GIS_8_0_1_released.md:35]:
```suggestion
    - ... further binary packages for other Linux distributions will follow shortly, please check at [software downloads](/download/software/index.html)
```

File: content/news/2022_02_24_GRASS_GIS_8_0_1_released.md (hunk 4 above)
Comment (nilason) [reply content/news/2022_02_24_GRASS_GIS_8_0_1_released.md:30]: You should remove it. No G8 on macports yet, PR stalled (for me) unknown reason (https://github.com/macports/macports-ports/pull/13801).

**Comment by neteler** [comment]
Nice:
https://grass.osgeo.org/news/2022_02_24_grass_gis_8_0_1_released/

**Pull request merged**